www/main.js

```diff
@@ -106,34 +106,35 @@ $(document).ready( function() {
                           enddate: endDate
                       }, function(eventHTML) {
                           $('#load-more').before(eventHTML);
+                          checkAnchors();
                       });
                       return false;
                  });
         });
     }
 
     function viewEvent(id) {
-        location.hash = 'event-' + id;
         container.empty()
             .append($('#show-all-template').html())
             .append($('#scrollToTop').html())
             .append($('#legend-template').html());
 
         getEventHTML({id:id}, function (eventHTML) {
             container.append(eventHTML);
+            checkAnchors();
         });
     }
-    
+
     function viewAbout() {
         var content = $('#aboutUs').html();
         container.empty().append(content);
+        checkAnchors();
```

Comment:
Unfortunate to have to do this after each dom change. How about $.on('click', 'a', clickFunction) ?